transports/quic/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "libp2p-quic"
+version = "0.6.0"
+authors = ["David Craven <david@craven.ch>", "Parity Technologies <admin@parity.io>"]
+edition = "2018"
+description = "libp2p-quic is a noise based quic implementation for rust-libp2p."
+repository = "https://github.com/ipfs-rust/libp2p-quic"
+license = "ISC"
```

Comment:
This is completely up to you (and I am not asking you to change this) however all other subcrates in libp2p are licensed under a MIT license.

Reply:
can change, ISC and MIT are interchangeable. as long as it's not gpl, I don't really care much about the license

Reply:
fixed in https://github.com/dvc94ch/rust-libp2p/pull/13